src/main/cpp/src/NativeParquetJni.cpp

```diff
@@ -93,6 +93,25 @@ struct column_pruning_maps {
   std::vector<int> chunk_map;
 };
 
+const int VALUE_TAG = 0;
+const int STRUCT_TAG = 1;
+const int LIST_TAG = 2;
+const int MAP_TAG = 3;
+
+const char * get_tag_name(const int tag) {
+  const char * tag_name = "unknown";
+  if (tag == VALUE_TAG) {
+    tag_name = "value";
+  } else if (tag == STRUCT_TAG) {
+    tag_name = "struct";
+  } else if (tag == LIST_TAG) {
+    tag_name = "list";
+  } else if (tag == MAP_TAG) {
+    tag_name = "map";
+  }
+  return tag_name;
+}
```

Comment:
Let return `std::string` instead:
```suggestion
std::string get_tag_name(const int tag) {
 switch(tag) {
  case VALUE_TAG: return "value";
  case STRUCT_TAG: return "struct";
  case LIST_TAG: return "list";
  case MAP_TAG: return "map";
  default: return "unknown";
 }
```